lib/endo.ex

```diff
@@ -169,7 +169,7 @@ defmodule Endo do
 
   defp list_tables(Ecto.Adapters.Postgres, repo, filters) do
     for table <- Postgres.list_tables(repo, filters) do
-      Postgres.to_endo(table, repo.config())
+      Postgres.to_endo(table, Keyword.put(repo.config(), :repo, repo))
```

Comment:
Super nitpicky but the augmentation of `repo.config()` could be done outside of the `for` loop